docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: "3.8"
+
+services:
+  server:
+    build: 
+      context: ./src/server
+    container_name: cross-catalog-search-server
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./src/server:/app
```

Comment:
That's strange! It's been working fine for me this whole time. I like this because it makes it possible to reload the application in the container automatically when I make a change in my local copy.